app/service/rest.py

```diff
@@ -167,14 +167,13 @@ def handle_integrity_error(exc):
         'duplicate key value violates unique constraint "{}"'.format(constraint) in str(exc)
         for constraint in {"services_name_key", "services_email_from_key", "services_normalised_service_name_key"}
     ):
+        dupe_name = (
```

Comment:
"dupe" also means to deceive or trick, is this not confusing?

Reply:
im pretty familiar with the use, especially given you've got the context from the constraint log line above, but happy to change it.